ips_list.yml

```diff
@@ -21,44 +21,44 @@
 L2_tcdm_hybrid_interco:
   commit: tags/pulpissimo-v1.0
 adv_dbg_if:
-  commit: 66e20f8ef3aae0198f36137ba28c26ff12b47eaa
+  commit: v0.0.1
 apb/apb2per:
-  commit: tags/pulpissimo-v1.0
+  commit: v0.0.1
 apb/apb_adv_timer:
-  commit: tags/pulpissimo-v1.0
+  commit: v1.0.2
 apb/apb_fll_if:
   commit: tags/pulpissimo-v1.0
 apb/apb_gpio:
-  commit: 730a9204dbb0d7057f45ef833d0a6e868c46107b
+  commit: 64-bit-rebased
 apb/apb_node:
-  commit: tags/pulpissimo-v1.0
+  commit: v0.1.1
 apb_interrupt_cntrl:
-  commit: tags/pulpissimo-v1.0
+  commit: v0.0.1
+axi/axi:
+  commit: v0.7.0
 axi/axi_node:
-  commit: tags/pulpissimo-v1.0
+  commit: v1.1.3
 axi/axi_slice:
-  commit: tags/pulpissimo-v1.0
+  commit: v1.1.4
 axi/axi_slice_dc:
-  commit: 5f889f887e58f6d5dadd79616b16e1a63381d569
-axi/axi_mem_if:
-  commit: 313d075cac65e960fddc8b93848aceda18eebeac
+  commit: v1.1.3
 timer_unit:
-  commit: tags/pulpissimo-v1.0
+  commit: v1.0.2
 common_cells:
-  commit: 0b8c10c21c9f810509bbd7bf86cfbdc4e6626c8e
+  commit: v1.13.1
 fpnew:
-  commit: v0.5.3
+  commit: v0.6.1
 jtag_pulp:
-  commit: dbg_dev
+  commit: v0.1
 riscv:
-  commit: b07b742ad211481c9585ddacdb8cb3a6174b7fe6
+  commit: 520ef0a418e3dc88d760fcb7092cdb15a81e724d
```

Comment:
This is a downgrade, but we want to update RI5CY anyway to the last version. Any idea how to fix the bug that requires the downgrade?

Reply:
I think someone is working on this bug.